### DISCUSSION TITLE
does not contain cmakelists.txt

### DISCUSSION THREAD

**Discussion by tokisuno** [polybar/polybar#2745] [Build Support]
![image](https://user-images.githubusercontent.com/85533116/174449105-d94b2554-4ac3-4722-8ebe-3fc1b65b0e81.png)

I keep on getting this when trying to compile from source. I use i3 on mint 20.3 so polybar isn't in my default repos.

**Comment by patrick96** [comment]
You need to clone polybar with `git clone --recursive https://github.com/polybar/polybar`
